extension/src/experiments/model/quickPick.ts

```diff
@@ -42,7 +42,9 @@ const getItemWithDescription = (
 ) => {
   const item = getItem(experiment, firstThreeColumnOrder)
   if (!experiment.checkpoints && experiment.displayNameOrParent) {
-    item.description = experiment.displayNameOrParent
+    item.description = `${experiment.commit ? '$(git-commit)' : ''}${
```

Comment:
Do we have control over the size? If not I prefer the second one, else if we can make the icon a little smaller it could look good.

Reply:
> Do we have control over the size? If not I prefer the second one, else if we can make the icon a little smaller it could look good.

Nope, only one size option :(